src/hooks/useSearch.js

```diff
@@ -138,7 +138,8 @@ export default function useSearch() {
 
     const res = await fetchResults(debouncedSearchQuery, pageInfo?.endCursor);
 
-    setSearchResults((prev) => uniqBy([...prev, ...res.nodes], (v) => v.id));
+    // eslint-disable-next-line no-unsafe-optional-chaining
```

Comment:
I resolved this differently in #43. What is the reasoning behind disabling the rule here as opposed to removing the optional chaining?